libpod/service.go

```diff
@@ -240,8 +307,8 @@ func (p *Pod) maybeRemoveServiceContainer() error {
 		if !canRemove {
 			return
 		}
-		timeout := uint(0)
 		logrus.Debugf("Removing service container %s", serviceCtr.ID())
+		timeout := uint(10)
```

Comment:
This is not quite correct I think, I don't think we allow setting the stop timeout for the service container but this should really just pass nil so the stop code will default to `StopTimeout()` which then defaults to 10.

Reply:
Agreed.  Will update with the next push but will wait for CI feedback.